src/TableFormatter.go

```diff
@@ -79,10 +79,10 @@ func (tableFormatter *TableFormatter) executeTextFSM(planet Planet, opts *Opts)
 	err := cmd.Run()
 	if err != nil {
 		message := "thrown from tableFormatter.format->exec pythonscript"
-		full := fmt.Sprintf("%s\n --- Additional info: %s\n", err, message)
-		planet.outputStruct.output = fmt.Sprintf("%s\n%s", planet.outputStruct.output, full)
+		errorstring := fmt.Sprintf("%s\n --- Additional info: %s\n", err, message)
+		planet.outputStruct.output = fmt.Sprintf("%s\n%s\n", planet.outputStruct.output, makeRed(errorstring))
```

Comment:
How about adding an error field to outputStruct on the planet for error messages. At the end of the execution you will have to mark only one chunk of text red instead of peppering the whole code base with calls to makeRed function.